chalice/.chalice/config.json

```diff
@@ -54,7 +58,9 @@
         "LAMBDA_REDUCER_FUNCTION_NAME": "dcp-matrix-service-reducer-predev",
         "DYNAMO_STATE_TABLE_NAME": "dcp-matrix-service-state-table-predev",
         "DYNAMO_OUTPUT_TABLE_NAME": "dcp-matrix-service-output-table-predev",
-        "S3_RESULTS_BUCKET": "dcp-matrix-service-results-predev"
+        "S3_RESULTS_BUCKET": "dcp-matrix-service-results-predev",
+        "MATRIX_VERSION": "56cf1ad",
+        "DYNAMO_CACHE_TABLE_NAME": "dcp-matrix-service-cache-table-predev"
```

Comment:
How did the cache table make a reappearance? I thought I'd seen it for the last time lol

Reply:
Ah, that was from me testing the Cache Invalidation work on `predev`